src/Input/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -15,32 +16,40 @@ export interface ToggleSwitchProps {
 }
 
 const ToggleSwitch: React.FunctionComponent<ToggleSwitchProps> = ({
+  active,
   defaultActive,
   iconOptions,
   onChange,
   className,
   ...defaultProps
 }: ToggleSwitchProps) => {
-  const [active, setActive] = React.useState(defaultActive);
-  const iconColor = active ? SecondaryColor.actionblue : SecondaryColor.grey;
+  const [innerActive, setInnerActive] = React.useState(defaultActive);
```

Comment:
Wonder if we can make innerActive the single source of truth for easier readability? 
e.g. `const [innerActive, setInnerActive] = React.useState(defaultActive || active);`